src/lib.rs

```diff
@@ -22,7 +22,38 @@
 //!
 //! You can even add multiple diagrams!
 //!
-//! To see it in action, go to the [demo crate](https://docs.rs/aquamarine-demo-crate/0.1.2/aquamarine_demo_crate/fn.example.html) docs.rs page.
+//! To see it in action, go to the [demo crate](https://docs.rs/aquamarine-demo-crate/0.1.3/aquamarine_demo_crate/fn.example.html) docs.rs page.
+//!
+//! ### Dark-mode
```

Comment:
Can we `include!()` this exact part so we don't need to keep copies in aquamarine itself and in demo crate in sync?

Reply:
Do you know the way?

Reply:
This is the way

Reply:
On the more serious note, you can keep two crates (both aquamarine and demo crate) in the same repo which will make sharing trivial

Reply:
I don't really want to share all the doc among them, they currently only share a few lines

Reply:
Well, you managed to incorporate the same typo twice. I think sharing these lines is benefitial to avoid such situations in future

Reply:
True, but making a file for each shared line sounds like maintenance hell. It's much easier to just fix those typos.

Reply:
Whatever you decide. It is you who is the mantainer after all.